### DISCUSSION TITLE
How to setup bottarga to set a user as MR reviewer?

### DISCUSSION THREAD

**Discussion by theliuk** [mia-platform/community#408] [General]
### Description

We'd like that all the MR opened by bottarga to update repos dependencies to have a particular user as reviewer. Can it be done? If so, how?

Thanks.

**Comment by fredmaggiowski** [comment]
Hi @theliuk this is something related to the internal Mia-Platform toolchain, I think the internal chat is more appropriate to this kind of discussion.
